### PR TITLE
 Update reclaim-ai extension

### PR DESCRIPTION
**Pull request by maximilianzuern**
## Description
Closes https://github.com/raycast/extensions/issues/8048

Command: **My Calendar**
- Added new action Reschedule.

reclaim-ai 2023-09-02 at 18 36 19

reclaim-ai 2023-09-02 at 18 36 35

---
Command: **Search Tasks**
- Added action Mark incomplete to archived tasks.
reclaim-ai 2023-09-02 at 18 34 33

- Added option 'Add 15min' to Add Time action.

reclaim-ai 2023-09-02 at 18 33 55

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/reclaim-ai/src/my-calendar.tsx [hunk 1]
```diff
@@ -43,6 +61,58 @@ const EventActionsList = ({ event }: { event: Event }) => {
...
+            }}
+          />
+          <Action
+            title="1 week"
+            onAction={() => {
+              RescheduleTask(String(event.calendarId), event.eventId, "NEXT_WEEK");
+            }}
+          />
+        </ActionPanel.Submenu>
+      )}
```
Comment (webergnr) [inline extensions/reclaim-ai/src/my-calendar.tsx:115]:
```suggestion
      {event.reclaimManaged === true && (
        <ActionPanel.Submenu title="Reschule Event" icon={Icon.ArrowClockwise}>
          <Action
            title="15min"
            onAction={() => {
              rescheduleTask(String(event.calendarId), event.eventId, "FROM_NOW_15M");
            }}
          />
          <Action
            title="30min"
            onAction={() => {
              rescheduleTask(String(event.calendarId), event.eventId, "FROM_NOW_30M");
            }}
          />
          <Action
            title="1hr"
            onAction={() => {
              rescheduleTask(String(event.calendarId), event.eventId, "FROM_NOW_1H");
            }}
          />
          <Action
            title="2hrs"
            onAction={() => {
              rescheduleTask(String(event.calendarId), event.eventId, "FROM_NOW_2H");
            }}
          />
          <Action
            title="4hrs"
            onAction={() => {
              rescheduleTask(String(event.calendarId), event.eventId, "FROM_NOW_4H");
            }}
          />
          <Action
            title="1 day"
            onAction={() => {
              rescheduleTask(String(event.calendarId), event.eventId, "TOMORROW");
            }}
          />
          <Action
            title="2 days"
            onAction={() => {
              rescheduleTask(String(event.calendarId), event.eventId, "IN_TWO_DAYS");
            }}
          />
          <Action
            title="1 week"
            onAction={() => {
              rescheduleTask(String(event.calendarId), event.eventId, "NEXT_WEEK");
            }}
          />
        </ActionPanel.Submenu>
      )}
```

File: extensions/reclaim-ai/src/my-calendar.tsx [hunk 2]
```diff
@@ -20,13 +20,31 @@ type EventSection = { section: string; sectionTitle: string; events: Event[] };
...
 
-  const { getEventActions } = useEvent();
+  const { getEventActions, handleRescheduleTask } = useEvent();
 
   const loadEventActions = async () => {
     const actions = await getEventActions(event);
     setEventActions(actions);
   };
 
+  const RescheduleTask = async (calendarId: string, eventId: string, reschedule: string) => {
```
Comment (webergnr) [inline extensions/reclaim-ai/src/my-calendar.tsx:30]:
```suggestion
  const rescheduleTask = async (calendarId: string, eventId: string, reschedule: string) => {
```

File: extensions/reclaim-ai/src/my-calendar.tsx [hunk 3]
```diff
@@ -20,13 +20,31 @@ type EventSection = { section: string; sectionTitle: string; events: Event[] };
...
     const actions = await getEventActions(event);
     setEventActions(actions);
   };
 
+  const rescheduleTask = async (calendarId: string, eventId: string, reschedule: string) => {
+    await showToast(Toast.Style.Animated, "Rescheduling event...");
+    try {
+      const executeReschedule = await handleRescheduleTask(calendarId, eventId, reschedule);
+      if (executeReschedule) {
+        showToast(Toast.Style.Success, `Rescheduled"${event.title}" successfully!`);
```
Comment (webergnr) [inline extensions/reclaim-ai/src/my-calendar.tsx:35]:
```suggestion
        showToast(Toast.Style.Success, `Rescheduled" ${event.title}" successfully!`);
```

**Review by webergnr — APPROVED** [review]
lgtm !

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**